chains/mainnet/noble.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  chainID: 'noble-1',
+  lcd: 'https://noble-api.polkachu.com',
+  gasAdjustment: 1.75,
+  gasPrices: { uosmo: 0.025 }, // <-- update this
+  prefix: 'noble',
+  coinType: '118',
+  baseAsset: 'uosmo', // <-- update this
+  name: 'Noble',
+  icon: 'https://station-assets.terra.money/img/chains/Noble.svg',
+  ibc: {
+    toTerra: 'channel-',   // <-- update this
+    fromTerra: 'channel-',   // <-- update this
```

Comment:
I don't believe there is an IBC connection from noble to terra yet. 
However Strangelove could easily open up a connection and run a relayer. 
CC: @ Jelena647.

Reply:
awesome, please let us know when complete.

Reply:
Will do. 
We should get to this next week.

Reply:
Done! We have a PR for the channel info on the cosmos chain-registry:
https://github.com/cosmos/chain-registry/pull/2085